### ISSUE TITLE
Can't use in Mail.app

### ISSUE THREAD

**Issue by matype** [0xType/0xProto#26]
## What

0xProto is recognized and listed on Mail.app, but can't be used.

<img width="864" alt="image" src="https://github.com/0xType/0xProto/assets/1401513/3f33053e-b8bb-4e11-9010-abba1691d878">

I consider this issue to be of low priority since 0xProto is a programming font.

## Ref

https://github.com/0xType/0xProto/issues/15

**Comment by VoilaNeighbor** [comment]
Can't use in JetBrains IDE, either, probably due to the naming. But I can cope with this because I mainly use Visual Studio and Visual Studio Code.

**Comment by capiet** [comment]
@masaakim you've tried the new IDE from JetBrains - Fleet, but the issue with naming occurs in their other IDEs - IDEA, GoLand, CLion etc. It's impossible to choose the font, because it isn't shown in the list

**Comment by masaakim** [comment]
@capiet Thank you. Yes, I was indeed using Fleet.
I've noticed a similar issue in the Intel One Mono repository. Apparently, TTF fonts might be compatible with JetBrains CLion, unlike OTF. Would you mind giving that a try?
https://github.com/intel/intel-one-mono/issues/57

**Comment by capiet** [comment]
> Apparently, TTF fonts might be compatible with JetBrains CLion, unlike OTF.
You are right, I've installed Intel One Mono via OTF files and cannot find it in the font list in the IDE, however after installing TTF the font was found.
![изображение](https://github.com/0xType/0xProto/assets/79054211/09b2ab91-1ba1-4a24-8c8b-73b4eebb182a)

**Comment by masaakim** [comment]
Great! Thanks for confirming that.

**Comment by masaakim** [comment]
@VoilaNeighbor https://github.com/0xType/0xProto/issues/26#issuecomment-1822404539

**Comment by matype** [comment]
I found we can use [Zx Proto](https://github.com/0xType/0xProto/pull/112) in Mail.app

<img width="726" alt="Image" src="https://github.com/user-attachments/assets/3ab58b93-dd61-4317-ab90-e5a88012bf74" />

**Comment by matype** [comment]
I've just released [v2.400](https://github.com/0xType/0xProto/releases/tag/2.400) that includes "Zx Proto" font files! https://github.com/0xType/0xProto/issues/26#issuecomment-2816594989
